packages/e2e/src/specs/overview/mc_version_check.e2e.ts

```diff
@@ -0,0 +1,40 @@
+import NavigationTabs from "../../page-objects/tabs.page"
+import settingsPage from "../../page-objects/settings.page"
+import { version } from "../../../../app/package.json"
+
+describe("Navigation Menu check ", () => {
+
+    before(async () => {
+      // Waiting for device connected through USB
+      await browser.executeAsync((done) => {
+        setTimeout(done, 10000)
+      })
+    })
+
+      it("Check Mudita Center version - app. vs. package.json", async () => {
+        
+        const appVersionFromJson = version
```

Comment:
Small thing but maybe it more clear if you change the variable name to `appVersionFromPackageJson`? Not forcing though.